.gitignore

```diff
@@ -3,3 +3,5 @@ npm-debug.log
 build
 .antwar
 
+.idea
```

Comment:
Nice to know that, because I was always under the impression that this is a good practice, because big projects are done like this ([Angular](https://github.com/angular/angular/blob/master/.gitignore), [React ](https://github.com/facebook/react/blob/master/.gitignore)) Will remove it.